yarn.lock

```diff
@@ -7984,11 +7972,6 @@ universalify@^2.0.0:
   resolved "https://registry.yarnpkg.com/universalify/-/universalify-2.0.0.tgz#75a4984efedc4b08975c5aeb73f530d02df25717"
   integrity sha512-hAZsKq7Yy11Zu1DE0OzWjw7nnLZmJZYTDZZyEFHZdUhV8FkH5MCfoU1XMaxXovpyW5nq5scPqq0ZDP9Zyl04oQ==
 
-unorm@^1.5.0:
```

Comment:
Turns out this functionality is available natively now :) https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/normalize